interoperability-layer-openhim/swarm.sh

```diff
@@ -1,52 +1,63 @@
 #!/bin/bash
 
-# Constants
-readonly ACTION=$1
-readonly MODE=$2
-
-COMPOSE_FILE_PATH=$(
-  cd "$(dirname "${BASH_SOURCE[0]}")" || exit
-  pwd -P
-)
-readonly COMPOSE_FILE_PATH
-
-# Import libraries
-ROOT_PATH="${COMPOSE_FILE_PATH}/.."
-. "${ROOT_PATH}/utils/config-utils.sh"
-. "${ROOT_PATH}/utils/docker-utils.sh"
-. "${ROOT_PATH}/utils/log.sh"
-
-verify_core() {
-  local start_time
-  start_time=$(date +%s)
-  until [[ $(docker service ls -f name=instant_openhim-core --format "{{.Replicas}}") == *"${OPENHIM_CORE_INSTANCES}/${OPENHIM_CORE_INSTANCES}"* ]]; do
-    config::timeout_check "${start_time}" "openhim-core to start"
-    sleep 1
-  done
-
-  local await_helper_state
-  await_helper_state=$(docker service ps instant_await-helper --format "{{.CurrentState}}")
-  until [[ "${await_helper_state}" == *"Complete"* ]]; do
-    config::timeout_check "${start_time}" "openhim-core heartbeat check"
-    sleep 1
+declare ACTION=""
+declare MODE=""
+declare COMPOSE_FILE_PATH=""
+declare UTILS_PATH=""
+declare mongo_services=()
+declare service_names=()
+
+function init_vars() {
+  ACTION=$1
+  MODE=$2
+
+  COMPOSE_FILE_PATH=$(
+    cd "$(dirname "${BASH_SOURCE[0]}")" || exit
+    pwd -P
+  )
+
+  UTILS_PATH="${COMPOSE_FILE_PATH}/../utils/"
+
+  mongo_services=(
+    "mongo-1"
+  )
+  if [[ "${NODE_MODE}" == "cluster" ]]; then
+    mongo_services=(
+      "${mongo_services[@]}"
+      "mongo-2"
+      "mongo-3"
+    )
```

Comment:
How about for i in {1..3}

Reply:
done in [c0d70ba](https://github.com/jembi/platform/pull/197/commits/c0d70ba7882b95805a6b144f3bbf99e7b60f6c01)